lib/tasks/scripts/add_permission_to_access_level.rb

```diff
@@ -80,4 +82,12 @@ def eligible?(user)
   def valid_permission_for_access_level?
     access_level && access_level[:default_permissions].include?(permission_name)
   end
+
+  def log_string(permission, user, resource)
```

Comment:
Can we use named arguments? Also, the `string` suffix is redundant. I'd be comfortable with just `def log`

Reply:
added some specs via `#create`